test/apps/carto-dynamic-tile/datasets.ts

```diff
@@ -102,10 +102,10 @@ export default {
   },
   raster: {
     source: rasterSource,
-    tableName: 'cartodb-data-engineering-team.jarroyo_raster.sdsc23_5_quadbin',
+    tableName: 'cartobq.public_account.temperature_raster_int8_new',
```

Comment:
I'm having trouble testing this, tried:

1. set access token
2. switch connection name from 'bigquery' to 'carto_dw'
3. disabled all other datasets

Then I get no errors, but no layer shown on screen – this sounds like a global layer, anything obvious I'm missing?

Reply:
Zoom into Prague :)